From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@chadelofson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

mgreenegit committed:
revise overview to make intent clear

mgreenegit committed:
Merge remote-tracking branch 'upstream/master' into mgreenegit-dscextension-0

mgreenegit committed:
Merge remote-tracking branch 'upstream/master' into mgreenegit-dscextension-0

mgreenegit committed:
Add information about default script.

mgreenegit committed:
formatting lists

mgreenegit committed:
add confidence to ARM template approach

mgreenegit committed:
explain why cmdlets are available

mgreenegit committed:
Merge remote-tracking branch 'upstream/master' into mgreenegit-dscextension-0

mgreenegit committed:
Merge remote-tracking branch 'upstream/master' into mgreenegit-dscextension-0

mgreenegit committed:
update examples

genlin committed:
update the article

mgreenegit committed:
move getting started section

mgreenegit committed:
dash replace asterisk for lists

neilpeterson committed:
[AKS] added k8s dashboard and changed cluster name across all docs

AshokKumar-P committed:
[ForumsTeam] Delete repeated word under the "Enable Traffic Manager" section 

Delete repeated word under the "Enable Traffic Manager" section.

neilpeterson committed:
updates

mgreenegit committed:
Merge remote-tracking branch 'upstream/master' into mgreenegit-dscextension-0

mgreenegit committed:
fix yaml issues and broken links

mgreenegit committed:
issues with header

motanv committed:
RDTask #10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

motanv committed:
RDTask #10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

bwren committed:
Update management solution - OMS rebrand

Commit:
adding link to spark/cosmos document

Commit:
moving to different spot

PatAltimore committed:
Add web tutorial

cherylmc committed:
linux

davidmu1 committed:
new multisite tutorial

christiankuhtz committed:
monitor alerts limit

christiankuhtz committed:
tweak

GitHubber17 committed:
Merge pull request #32869 from cherylmc/cert3

linux

PatAltimore committed:
Add editor feedback

Commit:
Merge pull request #32832 from bwren/la-oms-update

Update management solution - OMS rebrand

rolyon committed:
Updated author related to pull request 28700

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon0205master

Commit:
LiveFyre replaced by GitHub Issues.  Time for PM Dhruv Malik to get comfortable using GitHub.

Commit:
Merge pull request #32871 from christiankuhtz/patch-50

monitor alerts limit

PatAltimore committed:
Formatting changes

rolyon committed:
Small Acrolinx fixes to increase score

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon0205master

ktoliver committed:
Merge pull request #32870 from davidmu1/tutorialhostsites

new multisite tutorial

PRMerger10 committed:
Merge pull request #32876 from MightyPen/dhruv-author-vnet-dns-20180205-a

LiveFyre replaced by GitHub Issues.  Time for PM Dhruv Malik

ktoliver committed:
Merge pull request #32860 from PatAltimore/patricka-b2c-web-tutorials-master

Add B2C web tutorials

motanv committed:
RDTask #10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

PRMerger13 committed:
Merge pull request #32875 from rolyon/rolyon0205master

Rolyon0205master

neilpeterson committed:
remove sub id

PRMerger20 committed:
Merge pull request #32804 from motanv/feature/update_chaos_acom_article_chaos_target_filter

RDTask #10625124 : Update Chaos ACOM article with Chaos Target Filter…

GitHubber17 committed:
Merge pull request #32678 from neilpeterson/aks-qs-k8s-portal

[AKS] added k8s dashboard and changed cluster name across all docs

Commit:
quickfix02

Commit:
Merge pull request #32878 from MarkusVi/quickfix02

quickfix02

christiankuhtz committed:
typo

rolyon committed:
Updated author for RBAC pages

rolyon committed:
Updated author and Acrolinx fixes

PRMerger14 committed:
Merge pull request #32879 from christiankuhtz/patch-51

typo

JYOTHIRMAISURI committed:
adding a note

PRMerger10 committed:
Merge pull request #32888 from JYOTHIRMAISURI/master

adding a note

JYOTHIRMAISURI committed:
modified a note

Lisaco88 committed:
Merge pull request #32885 from rolyon/rolyon-0205b-master

Rolyon 0205b master

PRMerger10 committed:
Merge pull request #32889 from JYOTHIRMAISURI/master

modified a note

JYOTHIRMAISURI committed:
adding a note

PRMerger19 committed:
Merge pull request #32645 from genlin/master76

vpn-gateway-troubleshoot-vpn-point-to-site-connection-problems.md

JYOTHIRMAISURI committed:
minor update to webhooks article

PRMerger19 committed:
Merge pull request #32893 from JYOTHIRMAISURI/master

adding a note

nabhishek committed:
updated DMG -> Self-hosted ir

PRMerger15 committed:
Merge pull request #32894 from nabhishek/patch-17

updated DMG -> Self-hosted ir

v-jasowells committed:
Update BingVoiceOutput.md

PRMerger16 committed:
Merge pull request #32895 from v-jaswel/master

Update BingVoiceOutput.md

AshokKumar-P committed:
[ForumsTeam] Typo

Minor Typo Correction

Sogup committed:
Update backup-azure-backup-faq.md

PRMerger19 committed:
Merge pull request #32901 from Sogup/patch-32

Update backup-azure-backup-faq.md

Commit:
date

PRMerger15 committed:
Merge pull request #32904 from Blackmist/freshness

date

carolinacmoravia committed:
M22775: Possible typo: "Sparkl"

Hello, @mumian, 

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

JYOTHIRMAISURI committed:
minor updates to unified webhook and log

mayanknayar committed:
Azure-to-Azure DR with IP Retention

Adding documentation for retaining IP addresses with Azure-to-Azure DR
using Azure Site Recovery

Commit:
Merge pull request #32907 from carolinacmoravia/patch-604

M22775: Possible typo: "Sparkl"

JYOTHIRMAISURI committed:
fixing note format

PRMerger12 committed:
Merge pull request #32908 from JYOTHIRMAISURI/master

minor updates to unified webhook and log

MGoedtel committed:
Updated include syntax issue

JYOTHIRMAISURI committed:
updated aggregate-on image

GitHubber17 committed:
Fixed typo

GitHubber17 committed:
Merge pull request #32843 from Blackmist/fresh18

adding link to spark/cosmos document

itechedit committed:
Merge pull request #32909 from mayanknayar/master

Azure-to-Azure DR with IP Retention

Commit:
task 922: Add Mexico and Latvia POPs for Varizon

GitHubber17 committed:
Merge pull request #32911 from JYOTHIRMAISURI/master

updated aggregate-on image

PRMerger18 committed:
Merge pull request #32913 from dksimpson/dean-cdn

task 922: Add Mexico and Latvia POPs for Varizon

mgreenegit committed:
minor text change

Commit:
Merge pull request #32910 from MGoedtel/UpdateLAIncludes26

Updated include syntax issue

itechedit committed:
Update extensions-dsc-credentials.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger18 committed:
Merge pull request #32683 from AshokKumar-P/patch-17

[ForumsTeam] Delete repeated word under the "Enable Traffic Manager" …

Commit:
Merge pull request #32900 from AshokKumar-P/patch-18

[ForumsTeam] Typo

itechedit committed:
Merge pull request #32741 from mgreenegit/mgreenegit-dscextension-0

Update DSC Extension documentation

v-alje committed:
Merge pull request #32916 from MicrosoftDocs/FromPublicRepo

From public repo
